src/test/java/edu/kis/powp/jobs2d/events/SelectTestComplexCommandOptionListener.java

```diff
@@ -0,0 +1,22 @@
+package edu.kis.powp.jobs2d.events;
+
+import edu.kis.powp.jobs2d.command.TriangleFigureComplexCommand;
+import edu.kis.powp.jobs2d.drivers.DriverManager;
+import edu.kis.powp.jobs2d.magicpresets.FiguresJoe;
+
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
+public class SelectTestComplexCommandOptionListener implements ActionListener {
+
+    private DriverManager driverManager;
+
+    public SelectTestComplexCommandOptionListener(DriverManager driverManager) {
+        this.driverManager = driverManager;
+    }
+
+    @Override
+    public void actionPerformed(ActionEvent e) {
+        new TriangleFigureComplexCommand().createComplexCommand().execute(driverManager.getCurrentDriver());
```

Comment:
Execution can be performed by "Run command" option. Instead of executing the command directly, set it as a "current command".